File: Modul-3/Soal-1/PRAK301-2410817110016-MuhammadAzriRahman.py
a, b, c = map(int, input("Masukan 3 angka yang ingin diurutkan:").split())

if a > b:
    a, b = b, a
if a > c:
    a, c = c, a
if b > c:
    b, c = c, b

print(f"{a} {b} {c}")

File: Modul-3/Soal-1/PRAK301-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int a,b,c,temp;

    printf("Masukan 3 angka yang ingin diurutkan:");
    scanf("%d %d %d", &a,&b,&c);

    if (a > b) {
        temp = a;
        a = b;
        b = temp;
    }
    if (a > c) {
        temp = a;
        a = c;
        c = temp;
    }
    if (b > c) {
        temp = b;
        b = c;
        c = temp;
    }
    printf("%d %d %d", a,b,c);
}